staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -226,8 +226,8 @@ func parseConfig(configFileName string) (map[string]storage.Interval, error) {
 	for _, line := range lines {
 		slices := strings.Split(line, ",")
 		size := len(slices)
-		if size >= 3 {
-			m[strings.TrimSpace(slices[0])] = storage.Interval{Begin: strings.TrimSpace(slices[1]), End: strings.TrimSpace(slices[2])}
+		if size >= 4 {
+			m[strings.TrimSpace(slices[0])] = storage.Interval{Begin: strings.TrimSpace(slices[1]), End: strings.TrimSpace(slices[2]), Whitelist: strings.TrimSpace(slices[3])}
```

Comment:
Perhaps just use a constant "default" as the value of whitelist

Reply:
Updated